float64[] joints
float64[] cartesians
float64[] moves
float64[] swings
float64 grippers
float64[] nuclear
int32 framesID
int32 commandID
